components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.dblookup.ui/src/main/java/org/wso2/carbon/mediator/dblookup/DBLookupMediator.java

```diff
@@ -350,6 +375,21 @@ private void createCustomDataSource(OMElement pool) {
         }
     }
 
+    /**
+     * Get the value of the given key attribute of the element with QName
+     * @param pool
+     * @param qName
+     * @return key attribute value
+     */
+    private String getKey(OMElement pool, QName qName) {
+        OMElement ele = pool.getFirstChildWithName(qName);
+        if (ele != null) {
+            return ele.getAttributeValue(new QName(ATT_KEY));
+        } else {

```

Comment:
We can remove else block replacing in just with return. (No added advantage with it for runtime, just for clean code)